schema/rpc.fbs

```diff
@@ -82,19 +82,21 @@ table SettingsRequest {}
 table SettingsResponse {
     steam_vr_trackers: SteamVRTrackersSetting;
     filtering: FilteringSettings;
+    vrc_osc: VRCOSCSettings;
     model_settings: solarxr_protocol.rpc.settings.ModelSettings;
 }
 
 table ChangeSettingsRequest {
     steam_vr_trackers: SteamVRTrackersSetting;
     filtering: FilteringSettings;
+    vrc_osc: VRCOSCSettings;
     model_settings: solarxr_protocol.rpc.settings.ModelSettings;
 }
 
 table SteamVRTrackersSetting {
     waist: bool;
     chest: bool;
-    legs: bool;
+    feet: bool;
```

Comment:
can't wait for this to break the gui with the next `npm i`

Reply:
its not a breaking change on the wire, so assuming we are pinning the git commit in the GUI like we ought to be given that this is an unstable API that can break on each commit, then we will be fine.

If we are not pinning the version, then someone should make that change in the GUI.